CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.1.0 (2021-04-20)
+
+- Added modern theme styles to the progress indicator component. [#66](https://github.com/blackbaud/skyux-progress-indicator/pull/66)
```

Comment:
```suggestion
- Added modern theme styles to the waterfall progress indicator component. [#66](https://github.com/blackbaud/skyux-progress-indicator/pull/66)
```
Added it because the pull request [#66] indicates that it's for the waterfall progress indicator. If it's for both progress indicators, then please disregard, and make component plural.

Reply:
I think this is actually OK as is. If I remember correctly, it's a weird case where we have a single module but two distinct design patterns. So we split up the docs to display the design content separately. But if you look at the Development tab content, it's all the same except for the code samples. So I think that this is technically correct as-is.

Reply:
Thanks, John, for explaining. Makes sense. Alex, please disregard my comment.